### PR TITLE
bsd form submission - no error ui

### LINKED ISSUE #33: Hook up email signup form
Blocks #30

### PR DESCRIPTION
**Pull request by secretrobotron**
fixes #33

### CODE REVIEW

**Comment by secretrobotron** [comment]
@thisandagain r?

File: lib/routes.js [hunk 1]
```diff
@@ -19,7 +21,28 @@ module.exports = function (server) {
     method: 'POST',
     path: '/signup',
     handler: function (request, reply) {
-      reply.file('index.html');
+      var email = request.payload.email + '';
+      var optin = request.payload.optin + '';
+
+      var emailCheck = !!email.match(/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/i);
```
Comment (thisandagain) [inline lib/routes.js]: Let's import `Joi` as to keep this consistent with how we do email validation in the API / Login:
https://github.com/hapijs/joi#stringemailoptions

**Comment by thisandagain** [comment]
Wonderful! One small comment re: email validation, but otherwise this is great. R+

File: lib/routes.js (hunk 1 above)
Comment (secretrobotron) [inline lib/routes.js]: :+1:

**Pull request merged**